sympy/physics/mechanics/_system.py

```diff
@@ -0,0 +1,937 @@
+        ...     RigidBody)
+        >>> q = dynamicsymbols('q')
+        >>> qd = dynamicsymbols('q', 1)
+        >>> m, k, b = symbols('m k b')
+        >>> wall = RigidBody('W')
+        >>> system = System.from_newtonian(wall)
+        >>> bob = Particle('P', mass=m)
+        >>> bob.potential_energy = S.Half * k * q**2
+        >>> system.add_joints(PrismaticJoint('J', wall, bob, q, qd))
+        >>> system.apply_force(bob, b * qd * system.x, reaction_point=wall)
+        >>> system.form_eoms(LagrangesMethod)
+        Matrix([[-b*Derivative(q(t), t) + k*q(t) + m*Derivative(q(t), (t, 2))]])
+
+        We can also solve for the states using the 'rhs' method.
+
+        >>> system.rhs()
+        Matrix([
+        [               Derivative(q(t), t)],
+        [(b*Derivative(q(t), t) - k*q(t))/m]])
+
+        """
+        # KanesMethod does not accept empty iterables
+        loads = self.loads if self.loads else None
+        if issubclass(eom_method, KanesMethod):
+            self._eom_method = KanesMethod(
+                self.frame, self.q_ind, self.u_ind, kd_eqs=self.kdes,
+                q_dependent=self.q_dep, u_dependent=self.u_dep,
+                configuration_constraints=self.holonomic_constraints,
+                velocity_constraints=self.holonomic_constraints.diff(
+                    dynamicsymbols._t).col_join(self.nonholonomic_constraints),
+                forcelist=loads, bodies=self.bodies,
+                explicit_kinematics=False)
+        elif issubclass(eom_method, LagrangesMethod):
+            self._eom_method = eom_method(
+                Lagrangian(self.frame, *self.bodies), self.q, loads,
+                self.bodies, self.frame, self.holonomic_constraints,
+                self.nonholonomic_constraints)
+        else:
+            raise NotImplementedError(f'{eom_method} has not been implemented.')
+        return self.eom_method._form_eoms()
+
+    def rhs(self, inv_method=None):
+        """Returns equations that can be solved numerically.
+
+        Parameters
+        ==========
+
+        inv_method : str
+            The specific sympy inverse matrix calculation method to use. For a
+            list of valid methods, see
+            :meth:`~sympy.matrices.matrices.MatrixBase.inv`
+
+        Returns
+        ========
+
+        Matrix
+            Numerically solvable equations.
+
+        See Also
+        ========
+
+        sympy.physics.mechanics.kane.KanesMethod.rhs:
+            KanesMethod's rhs function.
+        sympy.physics.mechanics.lagrange.LagrangesMethod.rhs:
+            LagrangesMethod's rhs function.
+
+        """
+        return self.eom_method.rhs(inv_method=inv_method)
+
+    @property
+    def mass_matrix(self):
+        r"""The mass matrix of the system.
+
+        Explanation
+        ===========
+
+        The mass matrix $M_d$ and the forcing vector $f_d$ of a system describe
+        the system's dynamics according to the following equations:
+        $$M_d \dot{u} = f_d$$
```

Comment:
The [SymPy docstring guide](https://docs.sympy.org/latest/contributing/docstring.html#sample-docstring) uses `.. math::` for inline maths. Is using `$$...$$` equivalent?

Reply:
Addressed in 9d9a8c1cc3532150f0e4764d9c5a6016cc80a939